opa/models/course.py

```diff
@@ -0,0 +1,19 @@
+from odoo import fields, models, _
+
+
+class Course(models.Model):
+    _name = 'opa.course'
```

Comment:
keep class name, file name and model name similar

```suggestion
    _name = 'course'
```

Reply:
I changed the file names so they match their _name attribute.